### DISCUSSION TITLE
Tables with "Create" button functionality

### DISCUSSION THREAD

**Discussion by sebm** [twilio-labs/paste#1106] [Q&A] [answered]
There are lots of tables in Console with "Create" buttons overlaid on their headers, like so:

![Screen Shot 2021-01-22 at 17 55 39](https://user-images.githubusercontent.com/188161/105565515-0c120480-5cdc-11eb-86dd-47a003e6890d.png)

This pattern doesn't appear to be in Paste at present. Is this something Paste would implement? If not, what is the Paste-approved way of presenting a table of items that you can also add a new entry to?

**Comment by corinne-nagel** [comment, marked as answer]
Hi @sebm , sounds like this would be a create pattern https://paste.twilio.design/patterns/create

We will continue to add patterns planned here as well:https://paste.twilio.design/patterns

**Reply by sebm** [reply]
thanks!
